lib/auth/api.go

```diff
@@ -1241,6 +1256,21 @@ func (w *OktaWrapper) DeleteOktaAssignment(ctx context.Context, name string) err
 	return w.NoCache.DeleteOktaAssignment(ctx, name)
 }
 
+// CreateApp creates a new application resource.
+func (w *OktaWrapper) CreateApp(ctx context.Context, app types.Application) error {
+	return w.NoCache.CreateApp(ctx, app)
+}
+
+// UpdateApp updates an existing application resource.
+func (w *OktaWrapper) UpdateApp(ctx context.Context, app types.Application) error {
```

Comment:
Is there ever a situation where we actually need separate `Create` and `Update`, as opposed to just using some `Upsert` operation (that currently doesn't exist for `App`, for whatever reason)?

Reply:
As part of the Okta service it's not needed, but I can't account for other use cases.